tests/backend/test_postgres_db.py

```diff
@@ -158,3 +162,32 @@ async def test_retry_policy_allow_retry(postgresql_url, asyncio_loop):
 
             # Cancel the backend nursery
             nursery.cancel_scope.cancel()
+
+
+@pytest.mark.trio
+@pytest.mark.postgresql
+async def test_datetime(postgresql_url, backend_factory):
+    now = DateTime.now()
+
+    async with triopg.connect(postgresql_url) as conn:
+        await handle_datetime(conn, "timestamptz")
+
+        await conn.execute(
+            """
+            DROP TABLE IF EXISTS datetime;
+            CREATE TABLE IF NOT EXISTS datetime (
+                _id SERIAL PRIMARY KEY,
+                timestamp TIMESTAMPTZ
+            )"""
+        )
+
+        async with conn.transaction():
+            await conn.execute(
+                *Q("INSERT INTO datetime (_id, timestamp) VALUES (0, $timestamp)")(timestamp=now)
```

Comment:
no need for `Q` here
this is useful when building complex queries, here we can just do 
```python
execute("INSERT INTO datetime (_id, timestamp) VALUES (0, $1)", now)
```